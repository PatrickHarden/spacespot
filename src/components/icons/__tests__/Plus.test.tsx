import React from 'react'
import { mount } from 'enzyme'
import { mountToJson } from 'enzyme-to-json'

import Comp from '../Plus'

describe('Plus icon', () => {
  it('renders without crashing', async () => {
    const wrapper = mount(<Comp />)
    expect(mountToJson(wrapper)).toMatchSnapshot()
  })
})
