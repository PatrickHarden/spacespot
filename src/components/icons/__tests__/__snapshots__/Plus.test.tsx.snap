// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Plus icon renders without crashing 1`] = `
<Plus>
  <svg
    height="1em"
    viewBox="0 0 18 17"
    width="1em"
  >
    <g
      fill="none"
      fillRule="evenodd"
      stroke="#6CB9D5"
      strokeLinecap="round"
      strokeWidth={2}
    >
      <path
        d="M9 1.6v13.8M1.64 8.5h14.72"
      />
    </g>
  </svg>
</Plus>
`;
